### DISCUSSION TITLE
Wondered if it was possible to show users live location on the map, didn't see any examples of that.

### DISCUSSION THREAD

**Discussion by davebrewer** [visgl/react-google-maps#474] [Q&A]
Wondered if it was possible to show users live location on the map, didn't see any examples of that.
Thanks.

**Comment by mrMetalWood** [comment]
There is no "built in" way from the Google Maps API or this library that would allow to show the live location of the user directly.

A common approach is to use a `useGeoLocation` hook like this https://usehooks.com/usegeolocation or write your own basic version like this https://medium.com/jsexpert/react-be966e8a0145

Then use the resulting location to render and update a marker on the map.
